woocommerce/woocommerce-admin
# Checklist: Remove sideloaded images to reduce build size, take 2

Fixes #3716 
Supersedes #3719 

Current builds of woocommerce-admin include over 5mb of images that are ultimately sideloaded - downloaded from the plugin directory, and uploaded to the media library. These images drastically increase the size of the build and in-turn, inflate the size of WooCommerce core to over 12mb.

I attempted to run the images through a jpg optimizer, and they still weighed in at 2.7mb total - which to me feels still way too costly for what they are ultimately used for. Furthermore, any existing woo site will never use these images, since they are only leveraged during the Homepage creation step in the new site task list.

As such, the _new_ proposed solution was @jameskoster did some big trimming of the `other` industry jpg to `57kb`, and this PR uses that one image across all industries. The other change I made due to the dark image was to change the font color in the image block to be white.

/cc @peterfabian 

### Detailed test instructions:

- Disable Jetpack, otherwise the new post flow will be directed to WordPress.com and that kind of makes testing this more difficult
- Go to Orders > Help > Setup Wizard and click the Enable button for the task list:

![image](https://user-images.githubusercontent.com/22080/74476556-66fd7380-4e5e-11ea-9add-a33025925df7.png)

- Go back to the Woo dashboard, and click the Personalize Your Store step, and then the create homepage button:

![image](https://user-images.githubusercontent.com/22080/74476603-8399ab80-4e5e-11ea-8c56-16f4508760e5.png)

- Verify the newly created homepage looks like this regardless of industry chosen:

<img width="1516" alt="Edit Page ‹ bend outdoors — WordPress 2020-02-18 11-41-30" src="https://user-images.githubusercontent.com/22080/74772182-5ffea880-5244-11ea-9157-fa8fffcea7e1.png">

### Changelog Note:

Performance: Remove sideloaded images to save on build size
